Source/WebCore/page/SecurityPolicy.cpp

```diff
@@ -198,7 +198,7 @@ bool SecurityPolicy::allowSubstituteDataAccessToLocal()
 
 bool SecurityPolicy::isAccessAllowed(const SecurityOrigin& activeOrigin, const SecurityOrigin& targetOrigin, const URL& targetURL, const OriginAccessPatterns& patterns)
 {
-    ASSERT(targetOrigin.equal(SecurityOrigin::create(targetURL).ptr()));
+    ASSERT(targetOrigin.equal(*SecurityOrigin::create(targetURL).ptr()));
```

Comment:
`ASSERT(targetOrigin.equal(SecurityOrigin::create(targetURL)));` should work. If not,
`ASSERT(targetOrigin.equal(SecurityOrigin::create(targetURL).get()));`.